search-commons/src/test/java/no/unit/nva/search/models/IndexDocumentTest.java

```diff
@@ -3,21 +3,32 @@
 import static no.unit.nva.search.IndexingConfig.objectMapper;
 import static no.unit.nva.testutils.RandomDataGenerator.randomJson;
 import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static nva.commons.core.attempt.Try.attempt;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.hamcrest.core.StringContains.containsString;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.stream.Stream;
 import no.unit.nva.identifiers.SortableIdentifier;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
 class IndexDocumentTest {
 
+    public static Stream<IndexDocument> invalidConsumptionAttributes() {
```

Comment:
Most (if not all) of these tests should be testable from the handler

Reply:
I agree. The problem is that test coverage is not calculated  globally  but  per module. I think many of the tests would have been deleted if we could calculate test coverage globally.